raiden/utils/serialization.py

```diff
@@ -4,28 +4,36 @@
 from eth_utils import to_bytes, to_canonical_address, to_checksum_address, to_hex
 
 from raiden.transfer.merkle_tree import LEAVES, compute_layers
-from raiden.utils import typing
+from raiden.utils.typing import Address, Callable, Dict, List, Tuple, TypeVar
 
+T = TypeVar('T')
+RT = TypeVar('RT')
 
-def identity(val):
+KT = TypeVar('KT')
```

Comment:
For all these stuff I left comments in the other PR by Augusto: https://github.com/raiden-network/raiden/pull/3664

Can you perhaps answer them?

Reply:
Since these changes are in a different PR... i will just revert the changes in this one.

Reply:
I see. I am not saying it's a bad change. I am just trying to understand it better.